_sass/globals/grid.scss

```diff
@@ -7,6 +7,11 @@
   overflow-x: hidden;
   overflow-y: auto;
 
+
+  &.overflow-y-hidden {
```

Comment:
I'm not sure what this does, I removed it from the element and it doesn't seem to make a difference.